tasks/providers/http-01/wellknown.yml

```diff
@@ -0,0 +1,18 @@
+---
+#      # acme_data.challenge_data =>
```

Comment:
What is the purpose of this comment? If I had to hazard a guess, I would imagine it was left here during debugging?